pentasharp/Models/Entities/Notifications.cs

```diff
@@ -22,8 +22,8 @@ public class Notifications
         public int UserId { get; set; }
 
         [Required]
-        [ForeignKey("TaxiBooking")]
-        public int BookingId { get; set; }
+        [InverseProperty("TaxiBooking")]
+        public int? BookingId { get; set; }
```

Comment:
leave one space in between properties and remove data annotations since we agreed on not using them like this anymore

ctrl + a , ctrl + k + f should format the code